### ISSUE TITLE
Arm Support

### ISSUE THREAD

**Issue by pst1337** [Place1/wg-access-server#154]
Please add armv7 support for Raspberry pi devices.

**Comment by GoliathLabs** [comment]
@pst1337 we're publishing images for arm/v7 devices as well as other architectures over at our fork (https://github.com/freifunkMUC/wg-access-server/)

**Comment by disconn3ct** [comment]
[place1/wg-access-server:v0.4.6](https://hub.docker.com/r/place1/wg-access-server/tags)

Don't use latest and you won't have this problem :)

**Comment by GoliathLabs** [comment]
That's right. Nevertheless, this project has not had any (security) updates for quite some time. As mentioned above, we maintain an active fork with many additional features and regular dependency and security updates
